### ISSUE TITLE
Redesign for blocks screen

### ISSUE THREAD

**Issue by rodrigo-fm** [LemmyNet/jerboa#1712]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

I think the blocks screen could use a redesign. I was using the app lately and saw the profile screen with these 3 tabs on the top:

![image](https://github.com/user-attachments/assets/41c6627f-3625-4d2f-9295-41e6d95b95ea)

Currently, the blocks screen displays all three lists on the same screen, I think that using tabs will make it more consistent with the general app's design and can help with the user experience (even though I do not have much knowledge about what makes a good user experience).

![image](https://github.com/user-attachments/assets/799bc6e0-1689-4d38-b0ae-6ba22b2a2d24)
*Current blocks screen design*

I also think about displaying the count of blocked elements on each tab, the tabs texts would look something like this:

Users (3) | Communities (2) | Instances (0)

I can also display no number instead of (0) if you think it's better.

Let me know if you guys like this suggestion, I can code it myself. I believe that this change won't require too much refactoring since it's just about rearranging the displayed information.

**Comment by dessalines** [comment]
Agree, that sounds like a good idea. Ping us either here or in the jerboa dev chat if you need any help.

**Comment by rodrigo-fm** [comment]
> Agree, that sounds like a good idea. Ping us either here or in the jerboa dev chat if you need any help.

Thanks, @dessalines. I'll try to open the PR throughout the next week.
